Directory.Build.props

```
@@ -0,0 +1,11 @@
+<Project>
+    <PropertyGroup>
+        <Deterministic>true</Deterministic>
+        <TargetFramework>net8.0</TargetFramework>
+        <LangVersion>12.0</LangVersion>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>

```

```suggestion
        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
        <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
```
you forgot to enforce code style in build